src/app/loadout/loadout-ui/menu-hooks.tsx

```diff
@@ -57,6 +57,7 @@ export function useLoadoutFilterPills(
     extra?: React.ReactNode;
   } = {},
 ): [filteredLoadouts: Loadout[], filterPillsElement: React.ReactNode, hasSelectedFilters: boolean] {
+  // eslint-disable-next-line @eslint-react/hooks-extra/prefer-use-state-lazy-initialization
   const [selectedFilters, setSelectedFilters] = useState<Option<FilterPillType>[]>(emptyArray());
```

Comment:
Could this just be `useState<...>(emptyArray)`? Not that it matters much

Reply:
It could - I struggled with it a bit because emptyarray should be inlined and that's cheaper than conditionally executing a function